### PR TITLE
15th batch of RN bug doc text

### PR DESCRIPTION
**Pull request by stevsmit**
Bug doc text.

Preview: https://deploy-preview-42457--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes 

All OLM

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -546,7 +546,12 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 === Registry
 
 [id="ocp-4-10-olm"]
-=== Operator lifecycle
+=== Operator Lifecycle Manager
+
+[id="ocp-4-10-olm-arm-enhancements"]
+==== ARM support for Operatlor Lifecycle Manager
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:552]:
```suggestion
==== ARM support for Operator Lifecycle Manager
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1206,6 +1211,16 @@ This update corrects the accepted label specification so that the plug-in import
...
 
+* Occasionally, update events on clusters would happen at the same time that Operator Lifecycle Manager (OLM) modified an object from the lister cache. This caused concurrent map writes. This fix updates OLM so it no longer modifies objects retrieved from the lister cache. Instead, OLM modifies a copy of the object where applicable. As a result, OLM no longer experiences concurrent map writes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2003164[*BZ#2003164*])
+
+* Previously, Operator Lifecycle Manager (OLM) could not establish gRPC connections to catalog source pods that were only reachable through a proxy. If a catalog source pod was behind a proxy, OLM could not connect to the proxy and the hosted Operator content was unavailable for installation. This bug fix introduces a `GRPC_PROXY` environment variable that defines a proxy that OLM utilizes to establish connections to gRPC catalog sources. As a reuslt, OLM can now be configured to use a proxy when connecting to gRPC catalog sources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011927[*BZ#2011927*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:1216]:
```suggestion
* Previously, Operator Lifecycle Manager (OLM) could not establish gRPC connections to catalog source pods that were only reachable through a proxy. If a catalog source pod was behind a proxy, OLM could not connect to the proxy and the hosted Operator content was unavailable for installation. This bug fix introduces a `GRPC_PROXY` environment variable that defines a proxy that OLM uses to establish connections to gRPC catalog sources. As a result, OLM can now be configured to use a proxy when connecting to gRPC catalog sources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011927[*BZ#2011927*])
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -1206,6 +1211,16 @@ This update corrects the accepted label specification so that the plug-in import
...
+
+* Occasionally, update events on clusters would happen at the same time that Operator Lifecycle Manager (OLM) modified an object from the lister cache. This caused concurrent map writes. This fix updates OLM so it no longer modifies objects retrieved from the lister cache. Instead, OLM modifies a copy of the object where applicable. As a result, OLM no longer experiences concurrent map writes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2003164[*BZ#2003164*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:1218]: This is on line 1214 already :)

**Review by jeana-redhat — COMMENTED** [review]
small fixes~

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (stevsmit) [reply release_notes/ocp-4-10-release-notes.adoc:552]: lol

**Pull request merged**